docs/markup-language/README.md

```diff
@@ -1676,5 +1682,4 @@ endDate:Payment End Date;
 repeatEvery:"1 minute")]]
 ```
 
-*Important Note*: ERC-712 doesn't protect you from replay attacks out of the box. You will need to take it into account when you develop your smart contract. The fact that it doesn't protect is useful if you need to do recurring calls.
-
+_Important Note_: ERC-712 doesn't protect you from replay attacks out of the box. You will need to take it into account when you develop your smart contract. The fact that it doesn't protect is useful if you need to do recurring calls.
```

Comment:
Maybe expand on why the fact it doesn't protect is useful if you need to do recurring calls?

(It wasn't intuitive to me but it may be to others, but just in case)

Reply:
good point. Let me add to that